server/lib/chat/chat-service.ts

```diff
@@ -490,41 +519,48 @@ export default class ChatService {
     })
   }
 
-  async getChannelInfo(channelId: SbChannelId, userId: SbUserId): Promise<ChannelInfo> {
+  async getChannelInfo(channelId: SbChannelId, userId: SbUserId): Promise<GetChannelInfoResponse> {
     const channelInfo = await getChannelInfo(channelId)
+    // TODO(2Pac): Get this from DB instead.
+    const isUserInChannel = this.state.users.get(userId)?.has(channelId)
```

Comment:
Can you please just write this now instead of writing a TODO here, this isn't a hard query